### ISSUE TITLE
Removing a notification sound causes silence

### ISSUE THREAD

**Issue by UniversalSuperBox** [ubports/ubuntu-touch#892]
TL;DR: If we stop the `ubuntu-touch-sounds` package from installing to `/usr/share/sounds/ubuntu`, everyone who currently uses Ubuntu Touch (unless they don't do any customization at all) will not have ringtones or notification sounds.

## The problem

Right now, all preinstalled sounds in Ubuntu Touch are found in `/usr/share/sounds/ubuntu` (soon to be `/usr/share/sounds/ubports`). 

System software, such as system-settings (which sets the ringtone and notification tone), clock-app, and the date-time indicator, all hard-code the path where they will find a system-provided sound when saving the setting. They also will not fall back to their default sound if they are unable to load the user's chosen sound. This means two rather unfortunate things for us:

1. We cannot remove a sound, since it will cause a portion of our users to lose their set sound immediately
1. We cannot move the sound directory from `/usr/share/sounds/ubuntu/` since that would break all sounds for all users.

After a sound has been deleted, the user can of course set a new sound from System Settings without issue. However, that's not terribly obvious and their phones will be silent until they do.

As an interim solution for number 2, I've added a symlink from the old location to the new one in [ubuntu-touch-sounds](https://github.com/ubports/ubuntu-touch-sounds). It'd be best to not have to do this.

## Possible solution

I'm a bit surprised that this wasn't done already, since wallpapers and custom sounds already work like this.

We can introduce an update which copies the user's selected sounds from `/usr/share/sounds` into a directory in the user's home folder, such as `~/.config/ubuntu-system-settings/`. This allows us to remove sounds without disturbing users who already use that sound.

A couple of updates after introducing this setting (to make sure we get anyone who decides to skip an update), we can remove the symlinks from `/usr/share/sounds/ubuntu`.

**Comment by UniversalSuperBox** [comment]
A proper fix to this issue should also take into account #727, "Customization of ringtones". Probably not in the "add a custom ringtone for each contact" request, but in the "Add a custom notification sound at all" request.
